tethysext/atcore/services/resource_spatial_manager.py

```diff
@@ -77,35 +77,35 @@ def create_extent_layer(self, datastore_name, resource_id, geometry_type=None, s
         if geometry_type.lower() not in geometry_check_list:
             raise ValueError(f'{geometry_type} is an invalid geometry type. The type must be from one of the '
                              f'following: Polygon, LineString or Point')
-
-        # Get Default Style Name
-        default_style = f'atcore:{self.VL_EXTENT_STYLE}'
-
-        # feature name
-        feature_name = self.get_extent_layer_name(resource_id=resource_id)
-
-        sql_context = {
-            'resource_id': resource_id,
-            'srid': srid,
-        }
-
-        # Render SQL
-        sql_template_file = os.path.join(self.SQL_PATH, self.VL_EXTENT_VIEW + '.sql')
-        with open(sql_template_file, 'r') as sql_template_file:
-            text = sql_template_file.read()
-            template = Template(text)
-            sql = ' '.join(template.render(sql_context).split())
-
-        # Create SQL View
-        self.gs_api.create_layer(
-            workspace=self.WORKSPACE,
-            datastore_name=datastore_name,
-            feature_name=feature_name,
-            geometry_type=self.GT_POLYGON,
-            srid=srid,
-            sql=sql,
-            default_style=default_style,
-        )
+        else:
```

Comment:
This `else` is unecessary. The `raise ValueError` will prevent it from going on when needed.